rali/rali/include/cifar10_data_loader.h

```diff
@@ -39,6 +39,10 @@ class CIFAR10DataLoader : public LoaderModule
     void reset() override;
     void start_loading() override;
     std::vector<std::string> get_id() override;
+    std::vector<uint32_t> get_original_width() override;
```

Comment:
already defined decoded_image_info. Please use that instead of adding duplicate members